docs/userguide.md

```diff
@@ -82,43 +89,42 @@ Where `<image>` is in the form
 `<containerregistry>/<username>/<imagename>:<tagname>` e.g.
 `docker.io/<username>/federation-v2:test`.
 
+Any clusters listed after the call to the script will be joined to the federation, `cluster2` in this case.
+
 ## Manual Deployment
 
 If you'd like to understand what the script is automating for you, then proceed
 by manually running the commands below.
 
-### Deploy the Cluster Registry
+### Deploy Federation
 
-Make sure the storage provisioner is ready before deploying the Cluster
-Registry.
+First you'll need to create the namespaces to deploy the federation into:
 
 ```bash
-kubectl -n kube-system get pod storage-provisioner
+kubectl create ns federation
+kubectl create ns kube-multicluster-public
 ```
-
-Get version 0.0.4 of the [Cluster
-Registry](https://github.com/kubernetes/cluster-registry/releases/tag/v0.0.4) and run the
-following:
-
+Create a permissive rolebinding to allow federation controllers to run.
 ```bash
-crinit aggregated init mycr --host-cluster-context=cluster1
+kubectl create clusterrolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="federation:default"
```

Comment:
We should probably specify that this is temporary and will be made more restrictive in the future.